src/util/qrdisplay.js

```diff
@@ -6,12 +6,14 @@ const getQRDataURI = (data) => {
   return 'data:image/png;charset=utf-8;base64, ' + pngBuffer.toString('base64')
 }
 
-const openQr = (data) => {
+const openQr = (data, stop) => {
```

Comment:
maybe we should change the signature of these urilHandlers to use:

`{uri, stop}` then we can just pass the topic in and they can use stop if needed.